### PR TITLE
split gripper-servo-off and gripper-servo-on from certain method

### PR DESCRIPTION
**Pull request by knorth55**

### CODE REVIEW

File: jsk_2016_01_baxter_apc/euslisp/examples/picking-with-clustering.l [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 (defun place-object ()
   (send *ri* :move-arm-body->order-bin *arm*)
+  (send *ri* :gripper-servo-off arm)  ;; power off gripper servo not to overload it
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/examples/picking-with-clustering.l]: `(send *ri* :gripper-servo-off *arm*)`?

File: jsk_2016_01_baxter_apc/euslisp/examples/picking-with-sib.l [hunk 2]
```diff
@@ -18,6 +18,7 @@
 
 (defun place-object ()
   (send *ri* :move-arm-body->order-bin *arm*)
+  (send *ri* :gripper-servo-off arm)  ;; power off gripper servo not to overload it
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/examples/picking-with-sib.l]: (send *ri* :gripper-servo-off *arm*)?

**Comment by knorth55** [comment]
fixed

**Comment by pazeshun** [comment]
Can you also fix this?
https://github.com/start-jsk/jsk_apc/blob/master/jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l#L796-797
If you want to split gripper-servo-off from `:move-arm-body->order-bin`, it is better to split gripper-servo-on from `:spin-off-by-wrist`.
In master branch, picking-with-clustering.l, picking-with-sib.l and main.l need gripper-servo-on after `:spin-off-by-wrist`

**Comment by knorth55** [comment]
@pazeshun modified

File: jsk_2016_01_baxter_apc/euslisp/main.l [hunk 3]
```diff
@@ -213,6 +213,8 @@
           (send *ri* :wait-interpolation)
           (send *ri* :stop-grasp arm) ;; release object
           (send *ri* :spin-off-by-wrist arm :times 20)
+          (send *ri* :gripper-servo-on arm)
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/main.l]: In `return_object`, `move-arm-body->order-bin` doesn't exist, so this isn't needed.

File: jsk_2016_01_baxter_apc/euslisp/examples/picking-with-clustering.l [hunk 4]
```diff
@@ -22,6 +22,8 @@
   (send *ri* :wait-interpolation)
   (send *ri* :stop-grasp *arm*)
   (send *ri* :spin-off-by-wrist *arm* :times 20)
+  (send *ri* :gripper-servo-on *arm*)
+  ;; power on gripper servo as it was powered off in :move-arm-body->order-bin
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/examples/picking-with-clustering.l]: Please update the comment: `power on gripper servo as it was powered off in :move-arm-body->order-bin` -> `power on gripper servo`

File: jsk_2016_01_baxter_apc/euslisp/examples/picking-with-sib.l [hunk 5]
```diff
@@ -22,6 +22,8 @@
   (send *ri* :wait-interpolation)
   (send *ri* :stop-grasp *arm*)
   (send *ri* :spin-off-by-wrist *arm* :times 20)
+  (send *ri* :gripper-servo-on *arm*)
+  ;; power on gripper servo as it was powered off in :move-arm-body->order-bin
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/examples/picking-with-sib.l]: Maybe this comment is no more needed.

File: jsk_2016_01_baxter_apc/euslisp/main.l [hunk 6]
```diff
@@ -239,6 +241,8 @@
           (send *ri* :wait-interpolation)
           (send *ri* :stop-grasp arm)
           (send *ri* :spin-off-by-wrist arm :times 20)
+          (send *ri* :gripper-servo-on arm)
+          ;; power on gripper servo as it was powered off in :move-arm-body->order-bin
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/main.l]: Maybe this comment is no more needed.

**Comment by knorth55** [comment]
modified

**Comment by pazeshun** [comment]
+1

**Comment by pazeshun** [comment]
@knorth55 You need this to stow-task branch, don't you?

**Pull request merged**